OWML.SampleMods/OWML.LoadCustomAssets/LoadCustomAssets.cs

```diff
@@ -71,20 +75,29 @@ private void OnDuckLoaded(GameObject duck)
             duck.SetActive(false);
         }
 
-        private void OnEvent(MonoBehaviour behaviour, Events ev)
+        private void OnPlayerAwake(PlayerBody playerBody)
+        {
+            _playerBody = playerBody;
+            _playerTransform = playerBody.transform;
+        }
+
+        private void OnStartSceneChange(OWScene oldScene, OWScene newScene)
         {
-            if (behaviour.GetType() == typeof(PlayerBody) && ev == Events.AfterAwake)
+            if (oldScene == OWScene.SolarSystem)
             {
-                _playerBody = (PlayerBody)behaviour;
-                _playerTransform = behaviour.transform;
-                _isStarted = true;
-                ToggleMusic(ModHelper.Config.GetSettingsValue<bool>("enableMusic"));
+                _ducks.ForEach(Destroy);
```

Comment:
Must do this because custom loaded assets have DontDestroyOnLoad. In hindsight, that was a mistake, but probably too late to change now.